app/models/building.rb

```diff
@@ -14,6 +14,8 @@ class Building < ApplicationRecord
 
   has_one_attached :photo, dependent: :destroy
 
+  has_many :policies, as: :policy_makeable
```

Comment:
I think this needs to be removed. And shoudl building `include HasPolicies`

Reply:
Done